internal/genny/ci/ci_test.go

```diff
@@ -27,18 +27,36 @@ func Test_New(t *testing.T) {
 	r.Len(res.Files, 1)
 
 	f := res.Files[0]
-	r.Equal(".travis.yml", f.Name())
-	travisYml := struct {
-		Language     string
-		Go           []string
-		Env          []string
-		Services     []string
-		BeforeScript []string `yaml:"before_script"`
-		GoImportPath string   `yaml:"go_import_path"`
-		Install      []string
-		Script       string
+	r.Equal(".circleci/config.yml", f.Name())
+	circleYml := struct {
+		Version int
 	}{}
-	r.NoError(yaml.NewDecoder(f).Decode(&travisYml), ".travis.yml is a valid YAML file")
+	r.NoError(yaml.NewDecoder(f).Decode(&circleYml), "config.yml is a valid YAML file")
+}
+
+func Test_New_Github(t *testing.T) {
+	r := require.New(t)
+
+	g, err := New(&Options{
+		Provider: "github",
+		DBType:   "postgres",
+	})
+	r.NoError(err)
+
+	run := gentest.NewRunner()
+	r.NoError(run.With(g))
+	r.NoError(run.Run())
+
+	res := run.Results()
+	r.Len(res.Commands, 0)
+	r.Len(res.Files, 1)
+
+	f := res.Files[0]
+	r.Equal(".github/workflows/test.yml", f.Name())
+	githubYml := struct {
+		Name string
+	}{}
+	r.NoError(yaml.NewDecoder(f).Decode(&githubYml), "test.yml is a valid YAML file")
```

Comment:
I'm not sure if I should add something to this test?

Reply:
I think it could be fine if the yaml file format itself is fine.